tests/unit/test_sync.py

```diff
@@ -62,25 +62,7 @@ def test_get_files_from_wildcard(file, glob_files, result):
     "file,glob_files,result",
     [
         (
-            SyncFilesItem(src="file", dest="dest"),
-            None,
-            [SyncFilesItem(src="file", dest="dest")],
-        ),
-        (
-            SyncFilesItem(src=["file"], dest="dest"),
-            None,
-            [SyncFilesItem(src="file", dest="dest")],
-        ),
-        (
-            SyncFilesItem(src=["file1", "file2"], dest="dest"),
-            None,
-            [
-                SyncFilesItem(src="file1", dest="dest"),
-                SyncFilesItem(src="file2", dest="dest"),
-            ],
-        ),
-        (
-            SyncFilesItem(src="file/", dest="dest"),
+            SyncFilesItem(src="file/*", dest="dest"),
```

Comment:
Since `SyncFilesItem` still supports a list in `src`, these tests are still valid. Or am I missing something?

Reply:
hmm, it looks outdated...  You probably covered this situation in the following commits.